src/SFA.DAS.Commitments.Support.SubSite/assets/javascripts/search-index.js

```diff
@@ -0,0 +1,36 @@
+﻿(function ($) {
```

Comment:
The js in this file is duplicated in Search.cshtml and as far as I can tell, this js file isn't included directly (or in a bundle) in any cshtml/html/js file.
If that is actually the case, then we should either delete this file, or remove the script block in Search.cshtml and include this file.